model/clusters_mgmt/v1/addon_upgrade_policy_type.model

```diff
@@ -0,0 +1,39 @@
+/*
+Copyright (c) 2023 Red Hat, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// Representation of an upgrade policy that can be set for a cluster.
+class AddonUpgradePolicy {
+	// Cluster ID this upgrade policy is defined for.
+	ClusterID String
+
+	// Addon ID this upgrade policy is defined for
+	AddonID String
+
+	// Schedule cron expression that defines automatic upgrade scheduling.
+	Schedule String
+
+	// Schedule type can be either "manual" (single execution) or "automatic" (re-occurring).
+	ScheduleType String
+
+	// Upgrade type specify the type of the upgrade. Must be "ADDON".
+	UpgradeType String
```

Comment:
```suggestion
	// Upgrade type specifies the type of the upgrade. Must be "ADDON".
	UpgradeType String
```
What is the purpose of this field if it is always ADDON?

Reply:
Left over from when the addon upgrade policy was an upgrade policy.